### PR TITLE
use compressed indices setting up transmissibilities

### PR DESCRIPTION
**Pull request by akva2**
Extracted from https://github.com/OPM/opm-simulators/pull/2374

### CODE REVIEW

**Comment by akva2** [comment]
jenkins run norne norne_parallel please

File: ebos/ecltransmissibility.hh [hunk 1]
```diff
@@ -575,8 +575,8 @@ private:
 
                 auto isId = isId_(c1, c2);
 
-                int gc1 = std::min(cartMapper.cartesianIndex(c1), cartMapper.cartesianIndex(c2));
-                int gc2 = std::max(cartMapper.cartesianIndex(c1), cartMapper.cartesianIndex(c2));
+                int gc1 = std::min(c1, c2);
+                int gc2 = std::max(c1, c2);
```
Comment (bska) [inline ebos/ecltransmissibility.hh]: This doesn't look right to me, at least not in isolation.  The subsequent code block assumes that the `gc1` and `gc2` quantities are global (i.e., Cartesian) indices.  If they're not then the difference checks don't do what they're supposed to do.  We must either leave `gc1` and `gc2` intact or update the checks.

Comment (akva2) [reply ebos/ecltransmissibility.hh]: yes, this was overzealous. need to keep gc1/gc2, only change indexing below.

**Comment by akva2** [comment]
jenkins run norne norne_parallel please

**Comment by akva2** [comment]
okay to merge?

**Pull request merged**